### ISSUE TITLE
_Rx2.default is undefined

### ISSUE THREAD

**Issue by VinceOPS** [MCS-Lite/mcs-lite#481]
**PACKAGES**

**react-intl-cra** v0.2.9

node v8.6.0
create-react-app v1.4.1
react-scripts v1.0.17
React v16.2.0

**Code to reproduce the issue:**

Running the script (even without parameter) is enough to produce this error:
```
var srcPattern$ = _Rx2.default.Observable.of(srcPattern);

TypeError: Cannot read property 'Observable' of undefined
```

**Expected behavior:**
Works.

**Actual behavior:**
Does not work.

**Versions of packages used:** v0.2.9

**Fix**: 
Replace in `_interopRequireDefault`:
```
return obj && obj.__esModule ? obj : ...
```

With
```
return obj && obj.__esModule && obj.default ? obj : ...
```

**Comment by evenchange4** [comment]
It is a bug of Rxjs(https://github.com/ReactiveX/rxjs/issues/3155) and I have lock the version of rxjs (#482) as a workaround.

This should be fixed now on `react-intl-cra@0.2.10`. Thanks for reporting!

**Comment by evenchange4** [comment]
Close it. I moved `react-intl-cra` to [standalone repo](https://github.com/evenchange4/react-intl-cra). Please feel free to create issues there if the problem still occurs after `v0.2.12`.
